src/Microsoft.Extensions.DotnetToolDispatcher.Sources/project.json

```diff
@@ -2,7 +2,7 @@
   "version": "1.2.0-*",
   "shared": "*.cs",
   "dependencies": {
-    "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
+    "Microsoft.DotNet.Cli.Utils": "1.0.0-preview2-003121",
```

Comment:
They're all pre-release so when you say latest stable are you referring to https://www.nuget.org/packages/Microsoft.DotNet.Cli.Utils/1.0.0-preview2-1-003177?

The latest (preview3) is broken due to package not found issues:
```
Unable to resolve 'NuGet.Versioning (>= 4.0.0-rc-2048)' for '.NETStandard,Version=v1.6'.
[21:57:15][Step 3/4] Unable to resolve 'NuGet.Packaging (>= 4.0.0-rc-2048)' for '.NETStandard,Version=v1.6'.
[21:57:15][Step 3/4] Unable to resolve 'NuGet.Frameworks (>= 4.0.0-rc-2048)' for '.NETStandard,Version=v1.6'.
[21:57:15][Step 3/4] Unable to resolve 'NuGet.ProjectModel (>= 4.0.0-rc-2048)' for '.NETStandard,Version=v1.6'.
```

Let the CLI team know about it.